resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,44 @@
+define('wikia.flowTracking', ['wikia.tracker', 'mw'], function (tracker, mw) {
+
```

Comment:
I think yes, though each time at the moment a new flow tracking is added, not up-front